addons/actions/src/preview/__tests__/action.test.js

```diff
@@ -29,67 +28,66 @@ describe('Action', () => {
   });
 });
 
-// TODO: This functionality is removed, unsure if to add back or remove
```

Comment:
I saw that this test was commented out and covers the exact functionality that I was re-adding via this pull. Couldn't find the old implementation nor the reasoning for it's removal so if this was removed for a specific reason I would appreciate a pointer.

Reply:
Scratch that found the pull (https://github.com/storybooks/storybook/pull/4086/), but it is quite a large one so I'm still unable to glean the reason for the initial removal. Gonna change my implementation to mirror how user provided config options were merged with the config supplied by `configureActions`)